README.md

```diff
@@ -65,6 +65,15 @@ docker compose exec app ./vendor/bin/phpunit
 ##### phpMyAdmin
 phpMyAdmin is a PHP based web interface for MySQL databases. phpMyAdmin has been pre-configured to connect to and manage the Kos database for local debugging. To access the interface, navigate to [http://localhost:8001/](http://localhost:8001/) with Kos running.
 
+##### Commit hooks
+Commit hooks have been installed in this repository to perform checks prior to committing code back to the main code base. These hooks are installed and maintained into your local repository based on the configration maintained in this repository.
+
+First install `pre-commit`, follow the steps outlined in the Pre-Commit docs [https://pre-commit.com/](https://pre-commit.com/). This is usually as simple as `pip install pre-commit` or `brew install pre-commit`.
+
+Then install the hooks to your local repository by running `pre-commit install` from the base directory of the repository.
+
```

Comment:
I really hate leaving this kind of thing up to a developer to install and manage. I know pre-commit is great, but I wonder if there are any options that integrate directly with Composer. Something like Husky for node.

Another way I could see it working is on a composer install hook.

I worry about leaving more toil for the next person who comes along to work on this project.

Reply:
I hate the approach they take as well. At this moment in time, there's only a few project maintainers who likely have the hooks installed. Additional contributors will have code reviewed before merge. I've experimented in the past with using `pre-commit` hooks as a GitHub check but it's not perfect. I think for now something is better than nothing.

Reply:
> I think for now something is better than nothing.

I would disagree. I've found that less is often more in software. Without having a CI check in place and linting the repo first this is going to add noise to almost every PR.